tests/Feature/ContactTest.php

```diff
@@ -154,6 +224,51 @@ public function test_user_can_edit_a_gift_()
         );
     }
 
+    public function test_user_can_add_recipient_to_a_gift()
+    {
+        list($user, $contact) = $this->fetchUser();
+
+        $old_gift = factory(Gift::class)->create([
+            'contact_id' => $contact->id,
+            'account_id' => $user->account_id,
+        ]);
+
+        $otherContact = factory(Contact::class)->create([
+            'account_id' => $user->account_id,
+        ]);
+
+        $gift = [
+            'offered' => 'idea',
+            'name' => $this->faker->word,
+            'url' => $this->faker->url,
+            'value' => $this->faker->numberBetween(1, 2000),
+            'comment' => $this->faker->sentence(),
+            'has_recipient' => true,
+            'recipient' => $otherContact->id,
+        ];
+
+        $this->post(
+            '/people/'.$contact->hashID().'/gifts/'.$old_gift->id.'/update',
+            $gift
+        );
+
+        $gift = array_except($gift, ['offered', 'has_recipient', 'recipient']);
+
+        $this->assertDatabaseHas(
+            'gifts',
+            $gift + [
+                'is_an_idea' => true,
+                'has_been_offered' => false,
+                'contact_id' => $contact->id,
+                'account_id' => $user->account_id,
+                'is_for' => $otherContact->id,
+            ]
+        );
+
+        $new_gift = Gift::find($old_gift->id);
```

Comment:
`$new_gift` → `$newGift` to be PSR-4 compliant

Reply:
So, `old_gift` should be renamed with `oldGift` too?

Reply:
Indeed.